docs/source/1.0/spec/core/model.rst

```diff
@@ -1,223 +1,431 @@
-    shape_section :[`namespace_statement` [`use_section`] [`shape_statements`]]
-
-
-.. _namespaces:
-
-Namespaces
-==========
+---------------------------------
+Conventions used in this document
+---------------------------------
 
-Shapes are defined inside a :dfn:`namespace`. A namespace is mechanism for
-logically grouping shapes in a way that makes them reusable alongside other
-models without naming conflicts.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in :rfc:`2119`.
 
-.. _namespace-statement:
+This specification makes use of the Augmented Backus-Naur Form (ABNF)
+:rfc:`5234` notation, including the *core rules* defined in Appendix B
+of that document.
 
-Shapes can only be defined if a namespace is declared, and only a single
-namespace can appear in an IDL model file.
+Readers are invited to report technical errors and ambiguities in this
+specification to the Smithy GitHub repository at https://github.com/awslabs/smithy.
+This specification is open source, so contributions are welcome.
 
-.. productionlist:: smithy
-    namespace_statement :"namespace" `ws` `namespace` `br`
+.. rubric:: Examples
 
-The following example defines a string shape named ``MyString`` in the
-``smithy.example`` namespace:
+Unless declared otherwise, example Smithy models given in this specification
+are written using the :ref:`Smithy interface definition language (IDL) <idl>`
+syntax. Complementary :ref:`JSON AST <json-ast>` examples are provided
+alongside Smithy IDL examples where appropriate. For example:
 
 .. tabs::
 
     .. code-tab:: smithy
 
+        $version: "1.0"
+
+        metadata foo = "bar"
+
         namespace smithy.example
 
-        string MyString
+        use smithy.other.namespace#MyString
+
+        structure MyStructure {
+            @required
+            foo: MyString
+        }
 
     .. code-tab:: json
 
         {
             "smithy": "1.0",
+            "metadata": {
+                "foo": "bar"
+            },
             "shapes": {
-                "smithy.example#MyString": {
-                    "type": "string"
+                "smithy.example#MyStructure": {
+                    "type": "structure",
+                    "members": {
+                        "foo": {
+                            "target": "smithy.other.namespace#MyString",
+                            "traits": {
+                                "smithy.api#required": {}
+                            }
+                        }
+                    }
                 }
             }
         }
 
 
-.. _use-statement:
+.. _smithy-overview:
+
+---------------
+Smithy overview
+---------------
+
+Smithy is a framework that consists of a semantic model, file formats used to
+define a model, and a build process used to validate models and facilitate
+model transformations.
+
+.. figure:: images/smithy-overview.svg
```

Comment:
I would move away from using SVG, use PNG and consistent scaling as the changes in font size are distracting.

Reply:
With a little styling, text doesn't look bad...

Reply:
Updated to use text with some nice styling (via a custom Sphinx directive, no less!)